lib/CLI.js

```diff
@@ -80,7 +80,7 @@ Example:
 
       if (this.exit) {
         this._processExit = (exitStatus) => {
-          logger.verbose(`Exiting Dredd process with status '${exitStatus}'.`);
+          logger.debug(`Exiting Dredd process with status '${exitStatus}'.`);
```

Comment:
🤔 is this log message useful to anyone? Dredd exiting with a status code should already be pretty clear.

Reply:
Tried to improve this in 67b6b2e46ae4e272ef6011fbf0daa9032b42d8a3